config/prod.exs

```diff
@@ -16,7 +16,8 @@ config :logger, :console, level: :info
 
 config :realtime_signs,
   external_config_getter: ExternalConfig.S3,
-  restart_fn: &System.restart/0
+  restart_fn: &System.restart/0,
+  monitoring_api_key: System.get_env("MONITORING_API_KEY")
```

Comment:
This should be set in `runtime.exs` to make sure it picks up the runtime environment instead of the build environment.